### PR TITLE
5ika's First Goodie

### PR DESCRIPTION
**Pull request by 5ika**
**What does your Instant Answer do?**
My IA just introduces me :-P

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Because i've learn how to make good IA.

**What is the data source for your Instant Answer? (Provide a link if possible)**
No data source

**Why did you choose this data source?**

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**
 ``duckduckhack 5ika``

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
geeks

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
All others duckduckhack first goodies

**Are you having any problems? Do you need our help with anything?**

**Where did you hear about DuckDuckHack? (For first time contributors)**
On prism-break.org

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
ddh5ika

## Checklist
Please place an 'X' where appropriate.

```
[X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[X] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [X] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/5ika.pm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            twitter => "05Sika";
+
+# Triggers
+triggers start => "duckduckhack 5ika";
+
+# Handle statement
+handle remainder => sub {
+
+    return if $_;
+	return "Sika is awesome !";
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/5ika.pm]: Can you fix this hard-tab here? Should be 4 spaces :+1:

File: t/IsAwesome/5ika.t [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+use warnings;
+use Test::More;
+use DDG::Test::Goodie;
+
+zci answer_type => "is_awesome_5ika";
+zci is_cached   => 1;
+
+ddg_goodie_test(
+    [qw( DDG::Goodie::IsAwesome::5ika )],
+    # At a minimum, be sure to include tests for all:
```
Comment (mintsoft) [inline t/IsAwesome/5ika.t]: You can bomb out these comments here, they're just for instructional purposes

**Comment by mintsoft** [comment]
@5ika Congrats on your first goodie! I've just put a few small things to tweak against the diff, once they're sorted we'll merge it in and get it live!

**Comment by moollaza** [comment]
This LGTM :+1:

@5ika thanks a lot for contributing (we really appreciate it :smiley:), and congrats on making your first Goodie!

Now that you've made your first Goodie, we really hope you'll continue to contribute and make more Instant Answers! Please checkout our [Ideas forum](https://duck.co/ideas) if you're unsure about what to build. There's also lots of Github Issues in our repos that need someone to tackle them!

**Comment by moollaza** [comment]
@5ika congrats! It's live: https://duckduckgo.com/?q=duckduckhack+5ika&ia=answer

**Pull request merged**